### PR TITLE
PR

### PR DESCRIPTION
**Pull request by Zaq14rfv**
App Store的A和P应该大写

### CODE REVIEW

**Review by aboood506 — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -151,7 +151,7 @@ GBox软件源旨在构建一个ipa分享生态圈，采用json格式规范。制
     "appRepositories": [
         {
             # app类型，共有三种类型:
-            # SHAREING 分享类型，即直接提取app store得来的ipa，不能签名，直接安装，需要在app store登录对应的购买购买账号，即可正常使用，永不掉签
+            # SHAREING 分享类型，即直接提取App Store得来的ipa，不能签名，直接安装，需要在app store登录对应的购买购买账号，即可正常使用，永不掉签
             # SELF_SIGN 自签类型，需要签名后，才能正常安装，否则安装后图标为灰色不可用
             # ENT_SIGN 企业签类型，如果该企业签未被注销，则可直接安装，否则需要签名后安装
             # LINK  链接类型，直接打开网址，广告或宣传用途，需要填写appLink字段
```
Comment (Yazed550) [inline README.md:157]: http://sachcharak.com

**Review by Yazed550 — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (Sulxl) [reply README.md:157]: .

**Review by Sulxl — APPROVED** [review]

**Review by Sulxl — APPROVED** [review]

**Review by ammari24 — APPROVED** [review]

**Review by ammari24 — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -151,7 +151,7 @@ GBox软件源旨在构建一个ipa分享生态圈，采用json格式规范。制
     "appRepositories": [
         {
             # app类型，共有三种类型:
-            # SHAREING 分享类型，即直接提取app store得来的ipa，不能签名，直接安装，需要在app store登录对应的购买购买账号，即可正常使用，永不掉签
```
Comment (hefzym-Qagtaf-vamci0) [inline README.md:154]: AppleP12.com

**Review by hefzym-Qagtaf-vamci0 — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (abdoabdulfat) [reply README.md:154]: Hack

**Pull request closed without merging**